source/install/trouble_mysql.rst

```diff
@@ -0,0 +1,268 @@
+.. _mysql-installation-troubleshooting:
+
+MySQL Installation Troubleshooting
+----------------------------------
+
+Before you can run the Mattermost server, you must first install and
+configure a database. You can then start Mattermost by navigating to
+``/opt/mattermost/bin`` and entering the command ``sudo -u mattermost ./platform``.
```

Comment:
Specifying file paths and how to start the server seems unnecessary here since we're focused on the database here. The paths may also be incorrect depending on how Mattermost is installed (eg through the GitLab Omnibus or Docker)